app/client/src/features/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react';
+
+import { Button, Card, CardContent, Grid, Link, List, ListItem, Typography, IconButton } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { Add } from '@material-ui/icons';
+
+import { useRouter } from 'next/router';
+import { graphql, PreloadedQuery, usePreloadedQuery } from 'react-relay';
+
+import { DashboardEvent } from '@local/features/dashboard/DashboardEvent';
+import type { DashboardQuery } from '@local/__generated__/DashboardQuery.graphql';
+
+import { Loader } from '@local/components/Loader';
+
+const useStyles = makeStyles((theme) => ({
+    item: {
+        margin: theme.spacing(0, 0, 4, 0),
+    },
+    card: {
+        padding: theme.spacing(1, 1, 1, 1),
+    },
+    title: {
+        marginBottom: theme.spacing(1),
+    },
+    text: {
+        marginLeft: theme.spacing(1),
+    },
+    secondaryText: {
+        color: theme.palette.text.secondary,
+    },
+}));
+
+export const DASHBOARD_QUERY = graphql`
+    query DashboardQuery {
+        me {
+            events {
+                edges {
+                    node {
+                        id
+                        title
+                        description
+                        startDateTime
+                        endDateTime
+                        organization {
+                            name
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
+
+interface Props {
+    queryRef: PreloadedQuery<DashboardQuery>;
+}
+
+export function Dashboard({ queryRef }: Props) {
+    const data = usePreloadedQuery<DashboardQuery>(DASHBOARD_QUERY, queryRef);
+    const listOfEvents = React.useMemo(() => data.me?.events?.edges ?? [], [data.me]);
+    const classes = useStyles();
+    const router = useRouter();
+    const handleNav = (path: string) => () => router.push(path);
+    
+    if(!data) return <Loader />;
+
+    // Get current date to categorize events as current or upcoming
+    const d = new Date();
+    const currentDate = {
+        year: d.getFullYear(),
+        month: d.getMonth(),
+        day: d.getDate(),
+        time: d.getTime(),
+    };
+
+    // Store current events
+    // Would prefer to use `isActive` member of Event node, but this doesn't seem to have been implemented yet
+    const current_events = listOfEvents.filter(({ node: event }) => {
+        if (event.startDateTime && event.endDateTime) {
+            const eventStartDate = new Date(event.startDateTime.toString());
+            const eventEndDate = new Date(event.endDateTime.toString());
+            const eventStartTime = eventStartDate.getTime();
+            const eventEndTime = eventEndDate.getTime();
+            if (currentDate.year === eventStartDate.getFullYear() && 
```

Comment:
I don't think the year/month/day checks are necessary if you just compare date.getTime() you can see how it works in the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime)